CalibCalorimetry/EcalTrivialCondModules/src/EcalLaserCondTools.cc

```diff
@@ -233,14 +228,7 @@ void EcalLaserCondTools::processIov(CorrReader& r, int t1, int t2[EcalLaserCondT
            << "p2 = " << corr.p2 << "\t"
            << "p3 = " << corr.p3 << "\n";
     }
-    if (detid.subdetId() == EcalBarrel) {
-      if (corr.p1 < 0.55)
-        corr.p1 = .985;
-      if (corr.p2 < 0.55)
-        corr.p2 = .985;
-      if (corr.p3 < 0.55)
-        corr.p3 = .985;
-    }
```

Comment:
Are you removing these lines on purpose? If yes, please add few words about this change in the PR description.

Reply:
It's on purpose. PR description updated.